xcknife.gemspec

```diff
@@ -11,19 +13,19 @@ Gem::Specification.new do |s|
   s.homepage      = 'https://github.com/square/xcknife'
   s.licenses      = ['Apache-2.0']
 
-  s.summary       = %q{Simple tool for optimizing XCTest runs across machines}
+  s.summary       = 'Simple tool for optimizing XCTest runs across machines'
   s.description   = <<-DESCRIPTION
     Simple tool for optimizing XCTest runs across machines.
     Works by leveraging xctool's json-streams timing and test data.
   DESCRIPTION
 
-  s.files         = `git ls-files`.split($INPUT_RECORD_SEPARATOR).reject { |f| f =~ /^spec/} + ["TestDumper/TestDumper.dylib"]
+  s.files         = `git ls-files`.split($INPUT_RECORD_SEPARATOR).reject { |f| f =~ /^spec/ } + ['TestDumper/TestDumper.dylib']
   s.bindir        = 'bin'
-  s.executables      = ['xcknife', 'xcknife-min', 'xcknife-test-dumper']
+  s.executables = %w[xcknife xcknife-min xcknife-test-dumper]
   s.require_paths = ['lib']
   s.test_files    = s.files.grep(%r{^(test|spec|features)/})
 
-  s.required_ruby_version = '>= 2.0.0'
+  s.required_ruby_version = '>= 2.6.0'
```

Comment:
Rubocop doesn't support less than 2.4, and complains that this mismatches otherwise.